src/main/haskell/kore/src/Kore/Builtin/Krypto.hs

```diff
@@ -97,3 +111,147 @@ evalKeccak =
                 result = fromString (show digest)
             Builtin.appliedFunction
                 $ String.asExpandedPattern resultSort result
+
+evalECDSARecover :: Builtin.Function
+evalECDSARecover =
+    Builtin.functionEvaluator eval0
+
+eval0 :: Builtin.FunctionImplementation
```

Comment:
Please either move this implementation to a `where` clause, or give it a meaningful name. It is not immediately obvious that it has anything to do with `evalECDSARecover`.